.github/workflows/gh-pages.yaml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    branches:
+      - 'master'
```

Comment:
What's the reason for this?

Reply:
This is to run docs build on every PR. Step `deploy` that pushes build results is executed only on push.

Reply:
I understand, makes sense. Thanks :+1: